src/constants/common.js

```diff
@@ -5,13 +5,24 @@
  * Description : All commonly used constants through out the service
  */
 
-const successResponse = ({ statusCode = 500, responseCode = 'OK', message, result = [], meta = {} }) => {
+const FormsData = require('@db/forms/queries')
+const { InternalCache } = require('elevate-node-cache')
```

Comment:
@ankitpws we should be able to set the expiry time while requiring the package.
And move InternalCache and Redis to utils, you can write multiple functions to do the job.